docs/dashboard/app-templates-gallery.md

```diff
@@ -0,0 +1,53 @@
+---
+title: App Templates Gallery (preview)
+---
+
+Saleor can be extended with apps - and there are several ways how apps are built.
+
+App Templates Gallery is a part of Saleor Cloud Marketplace. As you can see, it doesn't contain exactly "apps",
+but rather templates ready to be spawned on your own instance.
+
+Saleor core is open source and we believe many our users will prefer their code self-hosted and adjusted.
+
+Quick comparison of apps differences:
+
+| App Template                          | 1st or 3rd party App                                          |
```

Comment:
Any chance we can do something about naming? Can we say "Templates" and "Apps"?

Reply:
In this context I think its fine to shorten them